### PR TITLE
Infra AWS Templates Group 5

### PR DESCRIPTION
**Pull request by brendalong**
# Summary

Migration of infra dashboards for
* amazon-aurora
* amazon-msk
* amazon-neptune
* aws/amazon-qldb
* amazon-rds-enhanced
* aws/amazon-rds
* aws/amazon-redshift
* aws/aws-nlb-alb

### Pre merge checklist
- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Are all documentation links publically accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [x] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/aws/aws-route-53/config.yml [hunk 1]
```diff
@@ -1,38 +0,0 @@
-id: bf043e5a-213c-411e-8eea-4e386a29a088
```
Comment (aswanson-nr) [inline quickstarts/aws/aws-route-53/config.yml:1]: Is this quickstart being intentionally deleted? I didn't see a replacement, so I wanted to check.

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Hey @brendalong, I found 2 things to take a look at, but overall this looks good!

File: quickstarts/aws/amazon-neptune/config.yml [hunk 2]
```diff
@@ -1,38 +1,39 @@
 id: 3ed7e509-9e82-4357-b076-82e30eeb0d1d
-slug: aws-neptunedb
+slug: aws-neptune
```
[the remark this answers is not in the document]
Comment (brendalong) [reply quickstarts/aws/amazon-neptune/config.yml]: 👍

File: quickstarts/aws/aws-route-53/config.yml (hunk 1 above)
Comment (brendalong) [reply quickstarts/aws/aws-route-53/config.yml]: Thanks for spotting this! We are moving this one to another group for validation and I should have replaced it.

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**